deploy/kubernetes/helm/che/custom-charts/che-plugin-registry/values.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+chePluginRegistry:
+  image: quay.io/openshiftio/che-plugin-registry
```

Comment:
We are moving to the new quay.io/eclipse organisation. Please use quay.io/eclipse/che-plugin-registry:nightly here.

Reply:
fixed